Library/Formula/scriptcs.rb

```diff
@@ -0,0 +1,25 @@
+require "formula"
+
+class Scriptcs < Formula
+  homepage "https://github.com/scriptcs/scriptcs"
+  url "https://github.com/scriptcs/scriptcs.git", :tag => "v0.13.3"
+  depends_on "mono" => :recommended
+
+  def install
+    script_file = "scriptcs.sh"
+    system "./build.sh"
+    libexec.install Dir["src/ScriptCs/bin/Release/*"]
+    (libexec/script_file).write <<-EOS.undent
+    #!/usr/bin/env bash
+    mono /usr/local/opt/scriptcs/libexec/scriptcs.exe $@
```

Comment:
Can't assume that Homebrew will always be in ` /usr/local `. Use ` #{Formula["scriptcs"].opt_libexec} ` instead.